spec/twitter/null_object_spec.rb

```diff
@@ -103,4 +103,10 @@
       expect(subject).not_to be_missing
     end
   end
+
+  describe "#presence" do
```

Comment:
the build is failing because we prefer single-quoted strings that don't require interpolation